documentation/element-api/client-server-rpc.asciidoc

```diff
@@ -4,22 +4,23 @@ order: 3
 layout: page
 ---
 
-ifdef::env-github[:outfilesuffix: .asciidoc]
 = Remote Procedure Calls
 
-RPC (Remote Procedure Calls) is a way of executing a procedure or subroutine in a different address space, typically on another machine.
+Remote Procedure Calls (RPCs) are a way to execute procedures or subroutines in a different address space, typically on another machine.
 
-Flow handles server-client communication allowing to invoke RPC from the server to client side and from the client to the server side.
+Vaadin Flow handles server-client communication by allowing RPC calls from the server to the client, and _vice versa_. 
 
-== Call a client-side method from the server side
+== Calling Client-side Methods from the Server
 
-It is possible to execute a client-side method from the server side by accessing the Element API.
+You can execute client-side methods from the server by accessing the `Element` API.
 
-The Element API `callFunction` allows executing a client-side component function taking as arguments the name of the function to call and the arguments to pass to the function.
+=== callFunction Method
 
-[NOTE]
-The arguments to pass to the function must be of a type supported by the communication mechanism.
-The supported types are `String`, `Boolean`, `Integer`, `Double`, `JsonValue`, `Element` and `Component`.
+The `callFunction` method allows you to execute a client-side component function on the server side. The method accepts two parameters: the name of the function to call, and the arguments to pass to the function. 
```

Comment:
```suggestion
The `callFunction` method allows you to execute a client-side component function from the server side. The method accepts two parameters: the name of the function to call, and the arguments to pass to the function. 
```

The element api is "proxy" to the client side, so we are really executing the method call only in the client side, just initiating the call from the server side code.